crates/experimentation_client/src/interface.rs

```diff
@@ -209,7 +211,7 @@ pub extern "C" fn get_satisfied_experiments(
 
     let local = task::LocalSet::new();
     let experiments = local.block_on(&Runtime::new().unwrap(), unsafe {
-        (*client).get_satisfied_experiments(&context)
+        (*client).get_satisfied_experiments(&context, None)
     });
     let experiments = match serde_json::to_value(experiments) {
         Ok(value) => value,
```

Comment:
```
let experiments = serde_json::to_value(experiments)
    .unwrap_or_else(|err| return error_block(err.to_string()));
```

Reply:
Can we make this change at all the places ? 
Ideally match statements should be used only for the cases where we have more than 2 conditions to match . 
For these cases , we can directly leverage map .